text/0000-variadic-generics.md

```diff
@@ -195,6 +195,15 @@ However, this approach introduces a lot of additional complexity. One of the
 complications is that such a trait couldn't be implemented for `()`, so
 there would have to be separate `Cons` and `Split` traits, rather than one
 unified `Tuple`.
+- Allow partial borrows or tuple reference splitting. Previous proposals have
+included a transformation from `&(Head, Tail...)` to `(&Head, &Tail...)`.
+However, this would require that every tuple `(Head, Tail...)` actually
+contain a sub-tuple `tail...`. This would get in the way of desirable
```

Comment:
> `(Head, Tail...)` would need the same repr `(Head, Tail)`

Is that true? Since the compiler is able to statically analyze the tuple'd types, it seems like it could be possible to have a policy that some type `(A, B, C)` had its head (`A`) at the beginning or end so long as the ordering is consistent across sub-tuples (i.e. `(B, C)` has the same repr in `(A, B, C)` as it does on its own, but `(A, B, C)` can be either `(A, (B, C))` or `((B, C), A)`).